website/server/controllers/api-v3/notifications.js

```diff
@@ -43,4 +43,44 @@ api.readNotification = {
   },
 };
 
+/**
+ * @apiIgnore Not yet part of the public API
+ * @api {post} /api/v3/notifications Mark notifications as read
+ * @apiName ReadNotifications
+ * @apiGroup Notification
+ *
+ *
+ * @apiSuccess {Object} data user.notifications
+ */
+api.readNotifications = {
+  method: 'POST',
+  url: '/notifications',
```

Comment:
This url should be changed to something else like `/notifications/read` because usually POST /notifications would mean a route to create a notification